terraform/aks/locals.tf

```diff
@@ -0,0 +1,7 @@
+locals {
+  environment                        = "${var.app_environment}-${var.app_suffix}"
+  review_additional_hostnames        = var.app_environment == "review" ? ["find-review-${var.app_suffix}.${var.cluster}.teacherservices.cloud"] : ["find-review-${var.app_suffix}.${var.cluster}.development.teacherservices.cloud"]
```

Comment:
review_additional_hostnames should only be used when var.app_environment == "review". Do we need a different condition here?

Reply:
I've updated this to use `env_config` and the main application to `use app_environment`